From f0e8b9ca41724ec0afd14c94d7b106a1d84d34b3 Mon Sep 17 00:00:00 2001
From: Alen Karnil <akarnil@witekio.com>
Date: Mon, 8 Apr 2024 15:00:51 +0100
Subject: [PATCH] curl-patch

---
 iotc-generic-c-sdk/CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/iotc-generic-c-sdk/CMakeLists.txt b/iotc-generic-c-sdk/CMakeLists.txt
index 03cf589..f8b61f2 100644
--- a/iotc-generic-c-sdk/CMakeLists.txt
+++ b/iotc-generic-c-sdk/CMakeLists.txt
@@ -43,12 +43,12 @@ ELSE ()
     target_link_libraries(iotc-c-generic-sdk paho-mqtt3cs)
 ENDIF ()
 
-IF (CMAKE_TOOLCHAIN_FILE)
-    # not the best way to detect VCPKG, but we'll go with that
-    find_package(CURL CONFIG REQUIRED)
-ELSE ()
-    find_package(CURL REQUIRED)
-ENDIF()
+# IF (CMAKE_TOOLCHAIN_FILE)
+#     # not the best way to detect VCPKG, but we'll go with that
+#     find_package(CURL CONFIG REQUIRED)
+# ELSE ()
+find_package(CURL REQUIRED)
+# ENDIF()
 
 target_link_libraries(iotc-c-generic-sdk CURL::libcurl)
 
